security/.htaccess

```diff
@@ -79,8 +67,19 @@ RewriteRule ^suites/secp256k1-2020/v1$ https://ns.did.ai/suites/secp256k1-2020/v
 RewriteRule ^suites/secp256k1-2019$ https://ns.did.ai/suites/secp256k1-2019 [R=302,L]
 RewriteRule ^suites/secp256k1-2019/v1$ https://ns.did.ai/suites/secp256k1-2019/v1 [R=302,L]
 
-
 # http://w3id.org/security/suites/x25519-2019
 
 RewriteRule ^suites/x25519-2019$ https://ns.did.ai/suites/x25519-2019 [R=302,L]
 RewriteRule ^suites/x25519-2019/v1$ https://ns.did.ai/suites/x25519-2019/v1 [R=302,L]
+
+# 🚧 DEPRECATED as of 04/29/2021 🚧
```

Comment:
This is maybe a silly complaint, but would it be ok to just use simple ASCII here?

Reply:
the point of the emoji is to make it clear there are deprecations, is reducing the visibility of deprecation warnings an enhancement to this PR?

Certainly if there is an encoding issue that this causes, I would remove it, but if this is just aesthetic preferences, I prefer to see deprecation warnings loud and clear.

Reply:
@davidlehn feel free to suggest changes, if lots of other folks prefer your aesthetic preferences, I am happy to accept them.

Reply:
I thought CAPITALIZATION was enough. :-)  And/or just some `################` delimiters.  I do have a path through some servers that has some console unicode issues, but I'll deal with the `?`s.

Reply:
Keeping in mind that not all emoji are visible on all systems (e.g., the `?`s displayed in some of @davidlehn's usage), I would recommend sticking with ASCII flags in places like this.  If you really want the construction barriers, you could probably find or recreate some ASCII art that delivers them. :-)